src/configure/utilities/webAppNodeVersionConverter.ts

```diff
@@ -0,0 +1,36 @@
+import { JSONPath } from 'jsonpath-plus';
+import * as vscode from 'vscode';
+import { ArmRestClient } from '../clients/azure/armRestClient';
+import { AzureSession } from "../model/models";
+import { Messages } from '../resources/messages';
+
+export async function webAppRuntimeNodeVersionConverter(nodeVersion: string, armUri: string, azureSession: AzureSession): Promise<string> {
+    if (nodeVersion.indexOf('|') >= 0){
+        nodeVersion = nodeVersion.split('|')[1];
+    }
+    if (nodeVersion === 'lts') {
+        let versions: string[];
+        await vscode.window.withProgress(
+            {
+                location: vscode.ProgressLocation.Notification,
+                title: Messages.GettingNodeVersion
+            },
+            async () => {
+                const resultSelector = "$.value[?(@.name === 'node')].properties.majorVersions[*].runtimeVersions";
+                const response = await new ArmRestClient(this.azureSession).fetchArmData(armUri, 'GET');
+                versions = JSONPath({ path: resultSelector, json: response, wrap: false, flatten: true });
+            }
+        );
+        let maxVersion = 0;
+        versions.forEach((version: string) => {
+            const match = version.match(/(\d+)-lts/);
+            if (match && match.length > 1) {
+                maxVersion = Math.max(maxVersion, +match[1]);
```

Comment:
Should it be match[0] ? I am not sure

Reply:
No, groups are stored from 1st index onwards. 0th contains the regex match.